qa/rpc-tests/test_framework/zip244.py

```diff
@@ -41,7 +41,7 @@ def transparent_digest(tx):
 def transparent_scripts_digest(tx):
```

Comment:
Add a TODO to update `transparent_digest` (just above here) with ZIP 244 changes.

Reply:
We already have an issue for this: #5446. We shouldn't make changes to the PR.